packages/app/src/views/Home.vue

```diff
@@ -9,11 +9,23 @@
       />
     </header>
     <main
-      class="h-2/3 overflow-x-hidden overflow-y-scroll bg-yellow-300 flex items-center justify-center text-4xl text-gray-800"
+      class="h-2/3 overflow-x-hidden overflow-y-scroll bg-yellow-300 flex flex-col text-4xl text-gray-800"
     >
+      <div class="w-full flex flex-col bg-gradient-to-b from-darkBlue to-darkViolet h-64">
+        <div class="text-white w-full text-xl h-1/5 flex items-center pl-10">
+          <span> Zuletzt gehört </span>
+          <Button round icon="fas fa-chevron-down" class="ml-auto mr-5 h-7 w-7 my-5" />
+        </div>
+        <div id="carousel" class="flex flex-row flex-grow content-start pt-5 overflow-x-scroll">
```

Comment:
I would suggest to use `overflow-x-auto` instead to allow the browser to hide the scrollbar in case there is no overflow. Maybe you could make the images even bigger / more margin between the tags so three would already make clear that you have to scroll. 

![Screenshot from 2021-01-12 11-20-14](https://user-images.githubusercontent.com/6918444/104301720-41db1000-54c8-11eb-9e5f-d85d6ea1c931.png)

Reply:
fixed in a7037769edd10cd00aff27d4e9346a0fbff9111e